web/server/codechecker_server/api/report_server.py

```diff
@@ -1745,6 +1760,16 @@ def getRunResults(self, run_ids, limit, offset, sort_types,
             filter_expression, join_tables = process_report_filter(
                 session, run_ids, report_filter, cmp_data)
 
+            annotation_keys = list(map(
```

Comment:
We needed to stand in front of a white table and look at examples to understand whats happening here. This is the part where we turn rows into columns, right? Could you please write about that a little?

Reply:
I added some docs.